tools/azure-pipelines/build-nightly-dist.yml

```diff
@@ -109,7 +109,7 @@ jobs:
             </settings>
             EOF
 
-            export CUSTOM_OPTIONS="-Dgpg.skip -Drat.skip -Dcheckstyle.skip --settings $(pwd)/deploy-settings.xml"
+            export CUSTOM_OPTIONS="-Dgpg.skip -Drat.skip -Dcheckstyle.skip -Dmaven.wagon.http.pool=false --settings $(pwd)/deploy-settings.xml"
```

Comment:
What is the reason for this change? Are we creating more connections than the pool can handle by default?

Reply:
Nobody knows: https://stackoverflow.com/a/58769870/568695 .. it's the black magic needed to make a very old maven version work on CI ;-)

Reply:
Alright, if it works... :-D

Reply:
I can not guarantee that it works, but at least in my dry run, it did: https://dev.azure.com/rmetzger/Flink/_build/results?buildId=9150&view=logs&j=eca6b3a6-1600-56cc-916a-c549b3cde3ff